signac/contrib/schema.py

```diff
@@ -332,6 +330,8 @@ def difference(self, other, ignore_values=False):
             )
         return ret
 
+    # TODO: We don't use this method anywhere that I see. Is there a particular use case that is
+    # facilitated by keeping it, or should we deprecate?
```

Comment:
It was probably used to implement this functionality at some point:
https://github.com/glotzerlab/signac/blob/c291e1ef1d8cc7bb3b9591ae4e6265f4c27cf7aa/signac/contrib/project.py#L923-L925

Reply:
I should have clarified. Yes, there is a test for it so coverage is green, but it's not used anywhere in the actual code base. I'm going to take this as approval for deprecating though :)